CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+# Contributing to xk6-browser
```

Comment:
It's pretty much a copy and paste from k6, with k6 replaced with xk6-browser in the appropriate places.